hs_composite_resource/models.py

```diff
@@ -254,6 +254,19 @@ def _recreate_xml_docs_for_folder(self, folder, check_metadata_dirty=False):
             for aggregation in aggregations:
                 aggregation.create_aggregation_xml_documents()
 
+    def get_aggregation_by_dataset_name(self, dataset_name):
+        """Get an aggregation that matches the aggregation dataset_name specified by *dataset_name*
+        :param  name: name (aggregation path) of the aggregation to find
```

Comment:
```:param name -> :param dataset_name```

Reply:
To match the comment, input parameter ````aggregation_name``` needs to be changed to ```dataset_name```

Reply:
Updated to aggregation_name